.github/workflows/publish.yaml

```diff
@@ -0,0 +1,33 @@
+name: publish
+on:
+  push:
+    branches:
+      - "master"
+      - "main"
+    tags:
+      - "v*"
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Login to image registry
+        uses: docker/login-action@v1
```

Comment:
How about we have commands like `make docker`,  `make docker-login`, and `make docker-push`. That way docker images can be built locally with ease and we can have simple commands in CI as well similar to how we do it in [observatorium/api](https://github.com/observatorium/api/blob/main/Makefile#L140).